src/NumericalAlgorithms/LinearOperators/IndefiniteIntegral.hpp

```diff
@@ -0,0 +1,148 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+/// \cond
+class DataVector;
+template <size_t>
+class Mesh;
+namespace gsl {
+template <typename T>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+// @{
+/*!
+ * \ingroup NumericalAlgorithmsGroup
+ * \brief Compute the indefinite integral of a function over the mesh.
+ *
+ * The integral is done from the lowest dim to the highest dim, and after each
+ * integral the boundary condition is such that the integral vanishes at the
+ * logical lower end of the domain. For example, integrating \f$x^2y\f$
+ *
+ * The integral is done from the lowest dim to the highest dim, and after each
+ * integral the boundary condition is such that the integral vanishes at the
+ * logical lower end of the domain. For example, integrating \f$x^2y\f$
```

Comment:
This paragraph is incomplete (last sentence truncated) and duplicated (same as previous paragraph).